app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split = command.split(" ")
```

Comment:
You can assign all your variables here!
`command, source..., dest... = command.split(" ")`